terraform-sentinel/intro.md

```diff
@@ -0,0 +1,19 @@
+[Sentinel](https://www.hashicorp.com/sentinel/) is the Policy-as-Code product from HashiCorp that enforces automated enforcement of logic-based policy decisions across all HashiCorp products.
+
+It allows customers to implement policy-as-code in the same way that Terraform implements infrastructure-as-code.
+
+The Sentinel Command Line Interface (CLI) allows you to apply and test Sentinel policies including those that use mocks generated from Terraform Cloud and Terraform Enterprise plans.
+
+This **Sentinel & Terraform Cloud** environment provides you a place to create and test policies for your infrastructure in **30 minutes**.
```

Comment:
```suggestion
This **Sentinel & Terraform Cloud** environment provides you a place to create and test policies against pre-populated mock data.
```

Reply:
Not sure if this was referring to the current Katacoda environment or to Sentinel embedded in TFC.